tock/tock/settings/test.py

```diff
@@ -6,6 +6,9 @@
 
 SECRET_KEY = get_random_string(50)
 
+# Needed for CG Django UAA to leverage internal cloud.fake service
+DEBUG = True
```

Comment:
Hmm, so this is a pretty big deviation from the [standard Django defaults](https://docs.djangoproject.com/en/2.0/topics/testing/overview/#other-test-conditions):

> Regardless of the value of the `DEBUG` setting in your configuration file, all Django tests run with `DEBUG=False`. This is to ensure that the observed output of your code matches what will be seen in a production setting.

Are there particular tests that will fail if `DEBUG` is `False`?  If there's only a handful, it might be useful to use `override_settings` for just the ones that need them... but if it'd make a lot of tests fail, it might be a sign that those tests aren't isolating the functionality to be tested as well as they could, and I could help with that.

Reply:
That's a great question and one I'm not sure either.  Thanks so much for pitching in, @toolness, I'm sure @rogeruiz would appreciate the help!